susemanager-utils/susemanager-sls/susemanager-sls.changes

```diff
@@ -1,3 +1,4 @@
+- Enable basic support for Ubuntu 22.04
```

Comment:
I don't see you are changing `susemanager-sls`, so this entry is not really needed.